src/Symfony/Bundle/FrameworkBundle/Command/DotenvDebugCommand.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Dotenv\Dotenv;
+
+/**
+ * A console command to debug current dotenv files with variables and values.
+ *
+ * @author Christopher Hertel <mail@christopher-hertel.de>
+ */
+final class DotenvDebugCommand extends Command
+{
+    protected static $defaultName = 'debug:dotenv';
+    protected static $defaultDescription = 'Lists all dotenv files with variables and values';
+
+    private $kernelEnvironment;
+    private $projectDirectory;
+
+    public function __construct(string $kernelEnvironment, string $projectDirectory)
+    {
+        $this->kernelEnvironment = $kernelEnvironment;
+        $this->projectDirectory = $projectDirectory;
+
+        parent::__construct();
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+        $io->title('Dotenv Variables & Files');
```

Comment:
```suggestion
        $io->title('Dotenv files with variables and values');
```

to take back what described upper :)

Reply:
i'd prefer the shorter title as well